lean4-infoview/src/infoview/info.tsx

```diff
@@ -50,20 +50,20 @@ export function InfoStatusBar(props: InfoStatusBarProps) {
         {isPinned && isPaused && ' (pinned and paused)'}
         <span className="fr">
             {copyGoalToComment &&
-                <a className="link pointer mh2 dim codicon codicon-quote"
+                <a id="copyGoalToComment" className="link pointer mh2 dim codicon codicon-quote"
```

Comment:
Technically it works because the HTML DOM defines getElementById such that it returns the "first" element with that id, but I will change it to a unique CSS class so I can use `getElementsByClassName('copytoComment')[0]` instead, would that be better?

Reply:
That does sound better, but then we are almost guaranteed to eventually forget these are for testing and use these classes to style things. How about `<a data-button="copyGoalToComment"` and then using a [dataset selector](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll#dataset_selector_attribute_selectors)?

Reply:
Done, I switched to dataset selector.  Thanks for the excellent suggestion.